cargo/lib/dependabot/cargo/update_checker/version_resolver.rb

```diff
@@ -237,6 +235,8 @@ def handle_cargo_errors(error)
             return nil
           end
 
+          raise Dependabot::DependencyFileNotResolvable, error.message if resolvability_error?(error.message)
```

Comment:
This is probably the right thing considering all existing specs pass but can't say I understand what `all possible versions conflict` actually means 😅

Reply:
I was hoping that a failing test would guide me. I quick dive into the history lead me to a PR where the rubocop rule for line length changed from 80 to 120. I didn't go deeper than that. I'll do a deeper dive and see what source of additional context that I can discover.